examples/server-communication__xhr-assertions/cypress/integration/multiple-requests.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+it('waits for multiple requests to finish', () => {
+  cy.visit('index.html')
+  cy.server()
+  cy.route('POST', '/posts').as('post')
+
+  // click both buttons - there will be 2 XHR requests going out
+  cy.get('#load').click()
+  cy.get('#delayed-load').click()
+
+  // there are two XHR calls matching our route
+  // wait for both to complete
+  cy.wait('@post').wait('@post')
+
+  // we can retrieve all matching requests using the following syntax
+  // cy.get('<alias>.all')
+  cy.get('@post.all').should('have.length', 2)
```

Comment:
A really great example of this is polling because this is a situation where you want to make sure it's not polling too much / too little, like:

```
cy.clock()
cy.visit('index.html')
cy.server()
cy.route('GET', '/items').as('getItems')
cy.wait('@getItems')
cy.tick(10000)
cy.wait('@getItems')
cy.get('@getItems.all').should('have.length', 2)
```

Reply:
ohh, I could tick in this recipe too - to avoid waiting full second!

Reply:
I added another spec to this example to show this in action - skipping the initial application's `setTimeout` delay via clock control